sbd_metrics.go

```diff
@@ -96,27 +89,24 @@ func readSdbFile() ([]byte, error) {
 }
 
 // retrieve a list of sbd devices from the config file contents
-func getSbdDevices(sbdConfigRaw []byte) ([]string, error) {
+func getSbdDevices(sbdConfigRaw []byte) []string {
 	// in config it can be both SBD_DEVICE="/dev/foo" or SBD_DEVICE=/dev/foo;/dev/bro
-	wordOnly := regexp.MustCompile("SBD_DEVICE=\"?[a-zA-Z-/;]+\"?")
-	sbdDevicesConfig := wordOnly.FindString(string(sbdConfigRaw))
-
-	// check the case there is an sbd_config but the SBD_DEVICE is not set
+	regex := regexp.MustCompile(`(?m)^\s*SBD_DEVICE="?((?:[\w-/];?)+)"?\s*$`)
```

Comment:
uh this regex looks complicated. Why did we changed it? perhaps we can add a comment of what he is doing. The previous one was simpler.

Reply:
Yeah, some commenting was indeed due!